packages/components/src/stories/PanelWithToolbar.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Button } from "@quri/ui";
+
+import { PanelWithToolbar } from "../components/ui/PanelWithToolbar/index.js";
+import { ToolbarItem } from "../components/ui/PanelWithToolbar/ToolbarItem.js";
+
+/**
+ * Internal UI component. Used in `SquigglePlayground`.
+ */
+const meta = { component: PanelWithToolbar } satisfies Meta<
+  typeof PanelWithToolbar
+>;
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Primary: Story = {
```

Comment:
Am I missing this? I don't see it in the Components Storybook.
![Uploading image.png…]()